src/camera_session.cpp

```diff
@@ -65,7 +81,7 @@ int CameraSession::init() {
       break;
 
     case CameraConfiguration::Adjusted:
-      break;
+      return 1;
```

Comment:
This should not be return 1, it's a valid case, it just means the config was adjusted

Reply:
It should break